src/nyc_trees/js/src/map.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+var L = require('leaflet');
+
+module.exports = {
+    create: create
+};
```

Comment:
@maurizi, I exported `create` this way because the module will probably have additional exports.